docs/model-dev-guide/dtrain/dtrain-introduction.rst

```diff
@@ -73,11 +65,12 @@ you have scaled your ``global_batch_size`` proportionally with ``slots_per_trial
 that training performance is being bottlenecked by network communication or disk I/O.
 
 To check if your training is bottlenecked by communication, we suggest setting
-``optimizations.aggregation_frequency`` in the :ref:`experiment-config-reference` to a very large
-number (e.g., 1000). This setting results in communicating updates once every 1000 batches.
-Comparing throughput with ``aggregation_frequency`` of 1 vs. ``aggregation_frequency`` of 1000 will
-demonstrate the communication overhead. If you do observe significant communication overhead, refer
-to :ref:`multi-gpu-training` for guidance on how to optimize communication.
+``optimizations.aggregation_frequency`` in the :ref:`experiment configuration
+<experiment-config-reference>` to a very large number (e.g., 1000). This setting results in
+communicating updates once every 1000 batches. Comparing throughput with an
+``aggregation_frequency`` of 1 vs. an ``aggregation_frequency`` of 1000 will demonstrate the
+communication overhead. If you do observe significant communication overhead, refer to
+:ref:`multi-gpu-training-implement-adv-optimizations` for guidance on how to optimize communication.
```

Comment:
@garrett361 please review

Reply:
resolved, i double checked the link the content to be sure one was supposed to reference the other